app/styles/core/variables.scss

```diff
@@ -0,0 +1,65 @@
+$goldenlight: #d1b147;
```

Comment:
Should we put this variables/mixins in one of the other variables files instead of creating a new one, or name/comment it so we know why there are multiple?
* https://github.com/codecombat/codecombat/blob/master/app/styles/style-flat-variables.sass
* https://github.com/codecombat/codecombat/tree/master/app/styles/mixins.sass
* https://github.com/codecombat/codecombat/blob/master/app/styles/bootstrap/_variables.scss